spec/features/reservations_spec.rb

```diff
@@ -549,7 +561,7 @@
   context 'valid items on confirmation page' do
     before(:each) do
       empty_cart
-      select('10', from: 'items_per_page') # show more than 1 item (default)
+      allow(@app_config).to receive(:override_on_create).and_return(false)
```

Comment:
Oh ok that makes sense, thanks for clarifying. Feel free to move it around if you want to, just let me know and I'll do one last review.